Buildings/Fluid/HeatExchangers/BaseClasses/WetCoilUARated.mo

```diff
@@ -0,0 +1,173 @@
+within Buildings.Fluid.HeatExchangers.BaseClasses;
+model WetCoilUARated
+  "This model calculates overall heat transfer coefficient, i.e., UA-value, from a rated condition for a cooling coil. 
+  Specify a rated condition either in fully-dry or fully-wet coil regime. Avoid that on partially-wet regime."
+  replaceable package MediumA=Buildings.Media.Air
+    constrainedby Modelica.Media.Interfaces.PartialMedium
+    "Air-side medium";
+  replaceable package MediumW=Buildings.Media.Water
+    constrainedby Modelica.Media.Interfaces.PartialMedium
+    "Water-side medium";
+
+  parameter Boolean use_UA_nominal = false
+    "Set to true to specify UA_nominal, or to false to use nominal conditions"
+    annotation (Evaluate=true,
+                Dialog(group="Nominal thermal performance"));
+  parameter Modelica.SIunits.Temperature TAirIn
+    "Air inlet temperature at a rated condition";
+  parameter Modelica.SIunits.Temperature TAirOut
+    "Air outlet temperature  at a rated condition";
+  parameter Modelica.SIunits.MassFraction wAirIn
+    "Absolute humidity of inlet air at a rated condition";
```

Comment:
This is ambiguous as one could think of humidity ratio in kg/kg_dryAir.
I already updated the description string for the corresponding parameter `X_w_a2_nominal` in `Fluid.HeatExchangers.WetCoilEffectivenessNTU`. It might be good to do the same for all humidity variables in this base class. And rename them using `X_w` notation.